tables/atom.py

```diff
@@ -366,8 +366,8 @@ def from_dtype(cls, dtype, dflt=None):
         warning is issued.
 
             >>> Atom.from_dtype(np.dtype('U20')) # doctest: +SKIP
-            Atom.py:392: FlavorWarning: support for unicode type is very limited,
-                and only works for strings that can be cast as ascii
+            Atom.py:392: FlavorWarning: support for unicode type is very
+                limited, and only works for strings that can be cast as ascii
```

Comment:
OK in principle thin could impact doctests but this particular test is SKIPed and doctests seems to be broken in general (see #874).
Probably it is better to include the change and think to broken doctests in a separate PR

Reply:
Oh, right, didn't take the doctests into account until you posted #874. How do actually doctests work when the line is too long for `flake8`?

Reply:
In my understanding doctest compares the output verbatim but I'm not sure what are indentation rules.
In any case there is the ``NORMALIZE_WHITESPACE`` directive that could help in this cases.